openeogeotrellis/geopysparkdatacube.py

```diff
@@ -1482,7 +1482,8 @@ def linear_scale_range(self, input_min, input_max, output_min, output_max) -> 'G
     def timeseries(self, x, y, srs="EPSG:4326") -> Dict:
         # TODO #421 drop old unsued "point timeseries" feature
         max_level = self.get_max_level()
```

Comment:
is this method still used? Maybe deleting it is better to reduce technical debt?